src/test/java/core/basesyntax/service/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+
+    @Test
+    public void writeToFile_validFilePath_Ok() throws IOException {
+        FileWriterService fileWriterService = new FileWriterServiceImpl();
```

Comment:
Make tested service class-level field - you use it in all methods.

**Same for other test classes.**